packages/dhis2/src/Adaptor.js

```diff
@@ -273,16 +223,23 @@ export function create(resourceType, data, options = {}, callback = s => s) {
       );
     } else {
       promise = request(configuration, {
-        method: 'post',
-        url: generateUrl(configuration, resolvedOptions, resolvedResourceType),
-        params,
+        method: 'POST',
+        path: generateUrlPath(
+          configuration,
+          resolvedOptions,
+          resolvedResourceType
+        ),
+        options: resolvedOptions,
         data: resolvedData,
-        ...requestConfig,
       });
     }
 
     return promise.then(result => {
-      const details = `with response ${JSON.stringify(result.data, null, 2)}`;
+      const details = `with response ${JSON.stringify(
+        result?.body?.message,
```

Comment:
common .request _always_ returns us a response object, so we don't need the question mark here. Also note that if `result` really is undefined, this function will throw a bit later down on `const { location } = result.headers;`